### ISSUE TITLE
Cannot update option template - time

### ISSUE THREAD

**Issue by nhatdq1810** [nhn/toast-ui.react-calendar#21]
## Version
1.0.4

## Test Environment
Browser: Chrome 77.0.3865.120
OS: Windows

## Current Behavior
* Set ```<Calendar ref={calendarRef} template={{time: getTimeTemplate(calendarView)}} />```
* When user changes calendar view - onChangeCalendarView even calls
```
calendarRef.current.getInstance().setOptions({
  template: { time: getTimeTemplate(calendarView) }
})
```
* Function getTimeTemplate is like
```
const getTimeTemplate = (calendarView) => (schedule) => {
  return calendarView === 'month' ? 'aaa' : 'bbb';
};
```
* The template of time is not changed

## Expected Behavior
The template should be changed

**Comment by jungeun-cho** [comment]
@nhatdq1810 
TOAST UI Calendar bug.
If you pass a template prop to setOptions, it doesn't apply!

I'll register the issue and manage it [here](https://github.com/nhn/tui.calendar/issues/487).

**Comment by nhatdq1810** [comment]
Many thanks @jungeun-cho

**Comment by jungeun-cho** [comment]
Patched in v[1.12.10](https://github.com/nhn/tui.calendar/releases/tag/v1.12.10) :)
